presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -798,9 +798,9 @@ The following arguments are available:
 ===================== ========== =============== =======================================================================
 Argument Name         required   type            Description
 ===================== ========== =============== =======================================================================
-``schema_name``       ✔️          string          Schema of the table to update
+``schema``            ✔️          string          Schema of the table to update
 
-``table_name``        ✔️          string          Name of the table to update
+``table``             ✔️          string          Name of the table to update
```

Comment:
Thanks for fixing this. One little thing, the argument `table` declared in `RollbackToSnapshotProcedure` has a problem when using named arguments as follows:

```
CALL iceberg.system.rollback_to_snapshot(schema => 'default', table => 'test_table', snapshot_id => 1383986162302017052)
```

In other procedures we declare the argument as `table_name` to avoid this problem. Do you mind modifying it in this PR to fix this problem and keep consistency with other procedures?

Reply:
Thanks, I updated it.